packages/twenty-front/src/modules/ui/field/display/components/LinksDisplay.tsx

```diff
@@ -1,50 +1,43 @@
 import { useMemo } from 'react';
 
+import { getFieldLinkDefinedLinks } from '@/object-record/record-field/meta-types/input/utils/getFieldLinkDefinedLinks';
 import { FieldLinksValue } from '@/object-record/record-field/types/FieldMetadata';
 import { ExpandableList } from '@/ui/layout/expandable-list/components/ExpandableList';
-import { checkUrlType } from '~/utils/checkUrlType';
 import {
   getAbsoluteUrlOrThrow,
   getUrlHostnameOrThrow,
   isDefined,
 } from 'twenty-shared/utils';
 import { LinkType, RoundedLink, SocialLink } from 'twenty-ui/navigation';
+import { checkUrlType } from '~/utils/checkUrlType';
 
 type LinksDisplayProps = {
   value?: FieldLinksValue;
```

Comment:
same here, value should be defined
(neat)

Reply:
As mentioned above, the value property must remain possibly `undefined`. This is also the case for the value of all the other field display components.

For example: 

![CleanShot 2025-05-20 at 11 50 08@2x](https://github.com/user-attachments/assets/bc31cfb2-fc6f-4639-9213-d1ce750d6f5a)